how-not-to-be-a-dinosaur.md

```diff
@@ -0,0 +1,117 @@
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_wtf1.png)  
+
+And at that time I was the only frontend developer in the company I work for. And I have always been the only frontend developer for all my previous employers. So there was nobody to convince me that it would be wise to embrace these new techniques.
+
+## Why I stood still for so long
+
+And late 2015, when I was already late to the party I got sick. A severe burnout and a big fallback one year later while I was still recovering. And when I felt a bit better, I came back to the office everything changed. There were two new frontend developers. With new repo's. New code. New words. New everything. 
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_wtf2.png)
+  
+
+It was 2017 already. I did not know where to start. I didn't dare to ask for help, because of my misplaced pride. I was writing code before some of my new colleagues were even born… and now I should go to these guys and admit I had no clue what I was doing?
+
+But I tried… I tried to keep up with it for a few months and then I was like: "I don't get it! fuck this shit!" 
+
+I have been doing design ánd development 50/50 for all of my carreer, so now it's time to say goodbye to one of these and focus on the other. And since I got só scared of The New Frontend World, I choose design. This was in early 2018.
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_drawing.png)
+
+_Nope. Not me either._
+
+And after a year of doing (UX) design, I found out this was the wrong choice. Years of intensive therapy and running into a black hole that was another potential burnout, I came to the conclusion that design was not for me. I can design, but I can't handle the stress that goes with it. 
+
+And then it was late 2018. I decided to get back to frontend development.
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_dino.png)
+
+_Yes this is me._
+
+## How to get up to speed? 
+
+Now I faced a big challenge. How to get up to speed after being paused for a few years in a world that changed (and still is changing) so fast? First thing to overcome was my aversion for The New Frontend World. I tried to look at React and all new tools with a fresh and unbiased view.
+
+While I was away in design world a lot has changed within our codebase, but also in The Frontend World. The tools we used improved or were replaced with better alternatives. The tools and codebase became more solid and mature.
+
+Then there was news. React Hooks was about to be released. Which made code só much more readable that I started thinking "Hmm, maybe I can do this".  
```

Comment:
```suggestion
Then there was news: React Hooks was about to be released. This made code _so_ much more readable that I started thinking "Hmm, maybe I can do this".  
```

Here I also think the reader would love to read slightly more about what made this difference so big for you. Especially because there has been, and still is, some backlash to the "magic" of hooks compared to the "clarity" and "obviousness" (to some people) of classes in React. 

Next to that, your revelation of "I can do this!" is a very big turnaround from your story before, for which you (rightfully) took your time. I think this is a key moment in the blog and deserves a bit more elaboration.

Reply:
You're right.. this deserves more attention. I'll try to add some Jip and Janneke explanation of hooks vs classes _and_ elaborate more on the revelation.